pkg/runtime/init.go

```diff
@@ -64,6 +66,20 @@ func (k *KubeadmRuntime) ConfigKubeadmOnMaster0() error {
 	return nil
 }
 
+func (k *KubeadmRuntime) handleKubeadmConfig() error {
+	//The configuration set here does not require merge
+	k.setCgroupDriver(k.getCgroupDriverFromShell(k.getMaster0IP()))
+	k.setKubeadmAPIAndCriSocketByVersion()
+	k.setInitLocalAPIEndpoint(k.getMaster0IP(), ApiServerPort)
```

Comment:
Using k.getApiServerPort()   Read it from kubeadmconfig, if not set return default port.

Reply:
Or set default port if initconfig not contains port feiled.